node_exporter/node_exporter.spec

```diff
@@ -27,11 +28,18 @@ Prometheus exporter for machine metrics, written in Go with pluggable metric col
 %install
 mkdir -vp %{buildroot}/var/lib/prometheus
 mkdir -vp %{buildroot}/usr/bin
-mkdir -vp %{buildroot}/usr/lib/systemd/system
 mkdir -vp %{buildroot}/etc/default
 install -m 755 node_exporter %{buildroot}/usr/bin/node_exporter
-install -m 644 %{SOURCE1} %{buildroot}/usr/lib/systemd/system/node_exporter.service
 install -m 644 %{SOURCE2} %{buildroot}/etc/default/node_exporter
+%if 0%{?el7}
```

Comment:
I'd reverse the logic and test for EL6 and then do the classic init. This should make it work on Fedora and prepare for EL8 which no doubt will have systemd.

Reply:
Updated